examples/webpack.config.js

```diff
@@ -42,26 +43,28 @@ module.exports = async (env) => {
 
     // see also production proxy at /examples/server/src/endpoints/proxy.js
     const proxy = {
-        '/gdc': {
-            target: backendUrl,
-            secure: false,
-            cookieDomainRewrite: '',
-            onProxyReq: (proxyReq) => {
-                console.log('proxy', '/gdc', proxyReq.path); // eslint-disable-line no-console
-                if (proxyReq.method === 'DELETE' && !proxyReq.getHeader('content-length')) {
```

Comment:
this check is not needed any more right ? :)

Reply:
I don't what it does, looks important. I moved it, it's not deleted.